framework/src/engine/legacy/constants.ts

```diff
@@ -11,6 +11,8 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
+
+export const BYTE_LENGTH = 4;
```

Comment:
this seems to be used for initializing buffer for int32, so name should reflect that